rune/libenclave/internal/runtime/pal/skeleton/liberpal-skeleton.c

```diff
@@ -120,7 +120,7 @@ static bool encl_create(int dev_fd, unsigned long bin_size,
 	secs->attributes = SGX_ATTR_MODE64BIT;
 	if (enclave_debug)
 		secs->attributes |= SGX_ATTR_DEBUG;
-	secs->xfrm = 7;
+	secs->xfrm = 3;
```

Comment:
Good catch. But we still need a more robust way to handle this issue, e.g, using cpuid to emulate whether AVX is available.